client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1172,7 +1172,7 @@ Undocumented.prototype.fetchSiteRecommendations = function( query, fn ) {
 };
 
 Undocumented.prototype.readRecommendationsStart = function( query, fn ) {
```

Comment:
Good call! I've [restored](https://github.com/Automattic/wp-calypso/pull/7778/commits/9483d285d6ae3394d545b102f7a07bdae677bdef) the readRecommendationsStart endpoint and [adjusted](https://github.com/Automattic/wp-calypso/pull/7778/commits/f0ba1465b2f5e1c0cd829c2af4b51f7e79ce01e0) the existing readRecommendedPosts function to accommodate other versions (like ours, v1.3!).